src/logic/community/members.ts

```diff
@@ -119,6 +121,27 @@ export async function createCommunityMembersComponent(
     }
 
     await communitiesDb.transferCommunityOwnership(communityId, targetAddress)
+    setImmediate(async () => {
+      const community = await communitiesDb.getCommunity(communityId)
```

Comment:
Two doubts here:
* Shouldn't we check if the community exists even before executing `await communityRoles.validatePermissionToTransferOwnership(communityId, updaterAddress, targetAddress)`? Another solution would to make that function to return community_id value in case it exists so we prevent going once again to the database
* Shouldn't we update the broadcaster to also notify this event `CommunityOwnershipTransferredEvent`?

Reply:
The entry point is the `updateMemberRole` function, which checks whether the community exists before calling the `transferOwnership` function, which is not publicly exposed.

I thought the same, but it's weird to call a broadcaster to send a notification to only one user. WDYT?